canonical-web-and-design/ubuntu.com
# Add extra copy and button to logged out /advantage

## Done

- Added free trial paragraph to logged out /advantage
- Added a button to a pre-filled product selector

## QA

- go to https://ubuntu-com-10006.demos.haus/advantage?test_backend=true
-  Click the button and see that it brings you to a modal where you can start the trial


## Screenshots

![image](https://user-images.githubusercontent.com/11927929/125066052-dbb7fa00-e0b2-11eb-8091-86717dc74a41.png)
